tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from synphot import SpectralElement\n",
+    "\n",
+    "agile = SpectralElement.from_file('agile.dat', wave_unit=\"nm\")\n",
```

Comment:
I think @eteq has some strong opinion about the API here. Erik, feel free to open a new issue to discuss it.